src/tzdata/TZData.jl

```diff
@@ -2,6 +2,14 @@ module TZData
 
 using Printf
 using ...TimeZones: DEPS_DIR
+if VERSION >= v"1.4"
+    using Pkg.Artifacts
+else
+    # to avoid fail when Julia tried to precompile even non-compatible code
+    macro artifact_str(name)
+        :(throw("this should never bee called"))
+    end
```

Comment:
Should be able to avoid having to define this if `@static` is used with `if VERSION` checks inside of functions

Reply:
Oh, that's what I was looking for, thanks! I thought something like that would be here but I could not find it.

Reply:
But we need to do this also in tests, thus not using `use_artifacts = VERSION >= v"1.4"`, right?